### DISCUSSION TITLE
¿Esto es normal en kata 0?

### DISCUSSION THREAD

**Discussion by Fernando-V** [LaunchX-InnovaccionVirtual/CursoIntroPython#182] [Q&A] [answered]
Hola, ando en kata 0, quise mirar sobre los repos pero por privado, pero al subir me sale todo el texto de código al instalar lo que me dice, solamente que no se si esta bien que salga o llegue a hacer algo mal, alguno me podría ayudar?, es que no entiendo si hice algo mal
![image](https://user-images.githubusercontent.com/99154039/153704621-b629dbe3-bc5c-4697-93bb-bc06cc0fecb8.png)

**Comment by OscarIslasH** [comment]
Si vas bien, el mensaje largo que te manda dice que se ha instalado correctamente el ipywidget
![Captura de pantalla 2022-02-12 042547](https://user-images.githubusercontent.com/99118114/153707543-6126f289-8577-4458-834e-4602d6dd3688.png)
s

**Reply by Fernando-V** [reply]
Entiendo muchas gracias por la aclaración

**Comment by Paulina-Ceciliano** [comment, marked as answer]
Si, es el mensaje que aparece porque descargaste un paquete y fue exitosa la instalación. Para borrar ese mensaje puedes darle clic en los puntitos que aparecen del lado izquierdo de la celda y luego clic en clear cell output.
![image](https://user-images.githubusercontent.com/88915301/153709845-1e633f9b-61db-44f3-b2fe-870005cbb603.png)

**Reply by Laura-Karina-GR** [reply]
Yo me trabé en ese punto (no puedo descargar el paquete), me puedes ayudar por favor? @Paulina-Ceciliano

**Reply by DiegoPenalozaJuarez** [reply]
ingresa la siguiente linea de codigo:

`pip install ipywidgets`

Si te marca error intenta con: 

`!pip install ipywidgets`

Te saldra un mensaje de ejecución largo, el chiste es que al final no te muestre un error y que se marque la palomita verde indicando que se ejecuto correcto la linea de codigo y se cargo la libreria. Ya despues sigue con los siguientes pasos del kata.

**Reply by Laura-Karina-GR** [reply]
Muchas gracias
